test/specs/lib/factories-test.tsx

```diff
@@ -524,18 +518,19 @@ describe('factories', () => {
 
     describe('from an element', () => {
       itReturnsAValidElement(<div />)
-      itAppliesDefaultProps(<div />)
       itDoesNotIncludePropsFromMappedProp(<div />)
-      itMergesClassNames('element', 'user', { value: <div className="user" /> })
       itAppliesProps('element', { foo: 'foo' }, { value: <div {...{ foo: 'foo' } as any} /> })
-      itOverridesDefaultProps(
-        'element',
-        { some: 'defaults', overridden: false },
-        { some: 'defaults', overridden: true },
-        { value: <div {...{ overridden: true } as any} /> },
-      )
-      itOverridesDefaultPropsWithFalseyProps('element', {
-        value: <div {...{ undef: undefined, nil: null, zero: 0, empty: '' } as any} />,
+
+      test('forwards original element "as is"', () => {
+        testCreateShorthand(
+          {
+            Component: 'p',
+            value: <span {...{ common: 'user', user: true } as any} />,
```

Comment:
may suggest `originalElement` to be used as value for `common` - then test (and, especially, its assertion) would be more descriptive